ada/views.py

```diff
@@ -33,9 +33,11 @@ def clickActivity(request: HttpRequest, pk: int, whatsappid: str) -> HttpRespons
     else:
         store_url_entry = RedirectUrlsEntry(symptom_check_url=redirect_url)
         store_url_entry.save()
+        customizationId = "kh93qnNLps"
```

Comment:
Can we add this to settings with this value as a default please?

Reply:
Alright.